gitpod-io/gitpod
# [PAPI] Retry steams upon not receiving data for 10 seconds

## Description

This change tries to improve our debugging situation with streams, because [streaming is hard](https://connectrpc.com/docs/go/streaming/#:~:text=comes%20with%20many-,drawbacks%3A,-It%20requires%20excellent).

## Related Issue(s)

Fixes ENT-753

## How to test

https://ft-debug-ub7eaf498b5.preview.gitpod-dev.com/workspaces

1. Make sure that prebuild status watching works (start a new prebuild)
2. Make sure that workspace status streaming does not time out (wait 10s on the workspace list page and see console)
